pkg/streamer/reader_test.go

```diff
@@ -289,8 +289,9 @@ func (t *testReaderSuite) TestParseFilerelayLogUpdatedOrNewCreated(c *C) {
 	}
 
 	// no valid update for relay sub dir, timeout, no error
-	ctx1, cancel1 := context.WithTimeout(context.Background(), 100*time.Millisecond)
+	ctx1, cancel1 := context.WithTimeout(context.Background(), time.Second)
 	defer cancel1()
+	t.createMetaFile(c, relayDir, filename, uint32(offset), notUsedGTIDSetStr)
```

Comment:
createMetaFile should be put together with `// create the current relay log file and write some events`